website/docs/r/security_center_subscription_pricing.html.markdown

```diff
@@ -21,13 +21,36 @@ resource "azurerm_security_center_subscription_pricing" "example" {
 }
 ```
 
+Example for extensions
+```hcl
+resource "azurerm_security_center_subscription_pricing" "test" {
+  tier= "Standard"
+  resource_type = "CloudPosture"
+  extension {
+    name= "SensitiveDataDiscovery"
+    enabled = "True"
+  }
+  extension {
+    name= "AgentlessDiscoveryForKubernetes"
+    enabled = "False"
+  }
+  extension {
+    name= "AgentlessVmScanning"
+    enabled = "True"
+    additional_extension_properties = {
+        ExclusionTags = "[]"
+    }
+  }
+}
+```
 ## Argument Reference
 
 The following arguments are supported:
 
 * `tier` - (Required) The pricing tier to use. Possible values are `Free` and `Standard`.
 * `resource_type` - (Optional) The resource type this setting affects. Possible values are `AppServices`, `ContainerRegistry`, `KeyVaults`, `KubernetesService`, `SqlServers`, `SqlServerVirtualMachines`, `StorageAccounts`, `VirtualMachines`, `Arm`, `Dns`, `OpenSourceRelationalDatabases`, `Containers`, `CosmosDbs` and `CloudPosture`. Defaults to `VirtualMachines`
 * `subplan` - (Optional) Resource type pricing subplan. Contact your MSFT representative for possible values.
+* `extension` - (Optional) Resource type pricing extensions. Contact your MSFT representative for possible values.
```

Comment:
```suggestion
* `extension` - (Optional) One or more `extension` blocks as defined below.
```

typically when we are documenting a property that is a block we will describe it like this in the documentation and then describe the properties within the block in another section below the top-level properties. [Here](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/website/docs/r/app_service_slot.html.markdown) is an example of some of our documentation with blocks references in them.

Reply:
Sue, updating